package.json

```diff
@@ -7,30 +7,31 @@
     "abstract-extension": "^3.0.1",
     "atomic-batcher": "^1.0.2",
     "bitfield-rle": "^2.2.1",
-    "bulk-write-stream": "^1.1.3",
     "codecs": "^2.0.0",
     "fast-bitfield": "^1.2.2",
     "flat-tree": "^1.6.0",
-    "from2": "^2.3.0",
     "hypercore-cache": "^1.0.1",
     "hypercore-crypto": "^2.0.0",
     "hypercore-protocol": "^8.0.5",
+    "hypercore-streams": "^1.0.1",
     "inherits": "^2.0.3",
     "inspect-custom-symbol": "^1.1.0",
     "last-one-wins": "^1.0.4",
     "memory-pager": "^1.0.2",
-    "merkle-tree-stream": "^3.0.3",
+    "merkle-tree-stream": "^4.0.0",
     "nanoguard": "^1.2.0",
     "nanoresource": "^1.3.0",
     "pretty-hash": "^1.0.1",
     "random-access-file": "^2.1.0",
     "sparse-bitfield": "^3.0.0",
+    "streamx": "^2.7.1",
     "timeout-refresh": "^1.0.3",
     "uint64be": "^2.0.1",
     "unordered-array-remove": "^1.0.2",
     "unordered-set": "^2.0.0"
   },
   "devDependencies": {
+    "buffer-from": "^1.1.1",
```

Comment:
needed?

Reply:
Yes, its needed for the tests. Previously it got installed with one of the removed libraries: https://github.com/hypercore-protocol/hypercore/blob/21e9e666bc182594172c4eb746cfbad6ffdb5ee8/test/streams.js#L4

Reply:
Nice, makes sense. Yea we should just move it to Buffer.from at some point